### DISCUSSION TITLE
Trying to update LCD display fw, but there is 3 versions on it

### DISCUSSION THREAD

**Discussion by Chrisyahya** [Jyers/Marlin#794] [Q&A] [answered]
I downloaded lcd fw from jyers github. But it confuses me..

I tried to search on wiki tab, seems like not yet documentation about "Gotcha" "Custom" "Original DWIN set Versions.

What are differences? i'm tempted to install gotcha version.
![image](https://user-images.githubusercontent.com/60070242/117860854-f8a29d80-b2ba-11eb-84c8-eb1a522a6bd9.png)

**Comment by gothcha** [comment]
Here are a few photos of the menus 
![IMG_5622](https://user-images.githubusercontent.com/80574048/117861279-2a027580-b289-11eb-9f2c-998c369ee2ab.jpeg)
![IMG_5625](https://user-images.githubusercontent.com/80574048/117861323-3686ce00-b289-11eb-9658-ede3f0470939.jpeg)
![IMG_5629](https://user-images.githubusercontent.com/80574048/117861375-46061700-b289-11eb-98b4-d1dae428adfe.jpeg)
![IMG_5627](https://user-images.githubusercontent.com/80574048/117861399-4d2d2500-b289-11eb-94a2-0b213e288605.jpeg)

**Reply by darco415** [reply]
nice

**Comment by Tuncay-Ayhan** [comment, marked as answer]
> I downloaded lcd fw from jyers github. But it confuses me..
> 
> I tried to search on wiki tab, seems like not yet documentation about "Gotcha" "Custom" "Original DWIN set Versions.
> 
> What are differences? i'm tempted to install gotcha version.
> ![image](https://user-images.githubusercontent.com/60070242/117860854-f8a29d80-b2ba-11eb-84c8-eb1a522a6bd9.png)

The Gotcha version has other icons. Just rename the folder to DWIN_SET , put it on a sd card. Put it into your display. You'll have to open it up. Then place the sd card in the slot. Put everything together and start your printer. Then you'll see a blue light, it will turn to orange in a couple of seconds. Orange means "I'm done flashing". Turn off the printer, remove the sd card from your screen and you are done! 


### TLDR
Just try them out. But if you want to flash one, don't forget to rename the folder to **DWIN_SET** and thén put it on your sd card.

**Comment by Cal-LF** [comment]
Try the Custom, they are the newest icons.

**Reply by smooreace** [reply]
I flashed both Custom and Gotcha. Definitely, no new icons involved or color changes. Both flash processes acted like they should... blue screen for a few seconds, the orange.

Do I have to build the icon set and insert it as well? or is that already done?

**Reply by Cal-LF** [reply]
I just tried both of the  DWIN_SET you did and both updated to their icon sets.
The screen reader is fussy about using the SD card formatted to 8gb and fat32 4096. That has been my finding.

**Comment by smooreace** [comment]
Finally got it with a default fat32 32k format. Not sure what was going on
before.

**Reply by Jyers** [reply]
Also btw, if you want the new icons and not just the upgraded ones you need to enabled DWIN_CREALITY_CUSTON_ICONS in your config

**Reply by Cal-LF** [reply]
@Jyers I do not have the custom icons setting turned on but still able to install the different icon packs. I did turn it on once to install icons but since then I have not turned it on in my compiles.

**Reply by Jyers** [reply]
It will let you install the icon packs without it enabled, you just won't get any additional icons such as the mesh icons. It's an easy thing to miss as without custom icons enabled these are simply replaced with existing icons.

**Reply by Cal-LF** [reply]
I now see those custom icons on the level menu.

**Comment by Byronstuff** [comment]
I am having some issues with the DWIN_SET process.  I am using a Macbook Pro, and can't figure out, for the life of me, how to get the DWIN files downloaded to my SD Card.  When I attempt to download the files, It just opens files deeper and deeper into the folder, never an option to actually download the file, or drag and drop on the FAT32 formatted SD card.

**Reply by Cal-LF** [reply]
You have to download the complete git zip file to your computer then you will have access to the display folder.
